gatling-maven-plugin/src/main/java/io/gatling/mojo/GatlingMojo.java

```diff
@@ -223,6 +229,29 @@ public void execute() throws MojoExecutionException {
     }
   }
 
+  private void iterateBySimulations(Toolchain toolchain, List<String> jvmArgs, List<String> testClasspath, Collection<String> simulations) throws Exception {
+    Exception exc = null;
+    for (String simulation : simulations) {
+      try {
+        executeGatling(jvmArgs, gatlingArgs(simulation), testClasspath, toolchain);
+      } catch (GatlingSimulationAssertionsFailedException e) {
+        if (simulations.size() == 1) throw e;
```

Comment:
Not exactly :)
for (int i = 0; i < simulations.size(); i++) {
try {
    executeGatling(jvmArgs, gatlingArgs(simulations(i)), testClasspath, toolchain);
  } catch (GatlingSimulationAssertionsFailedException e) {
    if (exc == null && i == simulations.size()) {
      throw e;
    }
}